Makefile

```diff
@@ -1,94 +1,71 @@
-DOCKER          = docker
-DOCKER_COMPOSE  = docker-compose
+SELF_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
+PHP_SERVICE := docker-compose exec php sh -c
 
 ##
 ## ----------------------------------------------------------------------------
-##   Setup
+##   Environment
 ## ----------------------------------------------------------------------------
 ##
 
-build: ## Build the environment
-	$(DOCKER_COMPOSE) pull --parallel --ignore-pull-failures
-	$(DOCKER_COMPOSE) build --pull
-
-env: ## Configure the environment variables
-	@if [[ ! -f docker-env ]]; then \
-		cp docker-env.dist docker-env; \
-	fi
-	nano docker-env
-
-start: ## Start the environment
-	@if [[ ! -f docker-env ]]; then \
-		echo 'The default configuration has been applied because the "docker-env" file was not configured.'; \
-		cp docker-env.dist docker-env; \
-	fi
-	$(DOCKER_COMPOSE) up -d --remove-orphans --no-recreate
-
-stop: ## Stop the environment
-	$(DOCKER_COMPOSE) stop
-
-restart: ## Restart the environment
-restart: stop start
-
-install: ## Install the environment
-install: build start
+apache: ## Open a terminal in the "apache" container
+	docker-compose exec apache sh
 
-uninstall: ## Uninstall the environment
-	$(DOCKER_COMPOSE) kill
-	$(DOCKER_COMPOSE) down --volumes --remove-orphans
+backup: ## Backup the "mysql" volume
+	docker run --rm \
+		--volumes-from $$(docker-compose ps -q mysql) \
+		-v $$(pwd):/backup \
+		busybox sh -c "tar cvf /backup/backup.tar /var/lib/mysql"
 
-.PHONY: build env start stop restart install uninstall
+build: ## Build the environment
+	docker-compose build
 
-##
-## ----------------------------------------------------------------------------
-##   Usage
-## ----------------------------------------------------------------------------
-##
+cache: ## Flush cache stored in Redis
+	docker-compose exec redis sh -c "redis-cli -n 1 FLUSHDB"
+	docker-compose exec redis sh -c "redis-cli -n 2 FLUSHDB"
 
-cache: ## Flush everything stored into the "redis" container
-	$(DOCKER_COMPOSE) exec -T redis sh -c "redis-cli FLUSHALL"
+composer: ## Install Composer dependencies from the "php" container
+	$(PHP_SERVICE) "composer install -o --working-dir=$(PROJECT_PATH)"
```

Comment:
It seem to me that PROJECT_PATH is undefined

Reply:
`PROJECT_PATH` as to be defined in the project Makefile as described in the [Wiki](https://github.com/EmakinaFR/docker-magento2/wiki/Makefile). But you're right, I'll add a comment to mention that. 😉